src/index.html

```diff
@@ -16,7 +38,403 @@
+                      src="./images/categories/smart-home-1.png"
+                      alt="Smart home img"
+                      class="category__photo"
+                    />
+                  </a>
+
+                  <a
+                    href="#"
+                    class="category__link categories__photo"
+                  >
+                    <img
+                      src="./images/categories/smart-home-2.png"
+                      alt="Smart home img"
+                      class="category__photo"
+                    />
+                  </a>
+                </div>
+
+                <h3 class="categories__title">Portable speakers</h3>
+              </article>
+              <div class="categories__button">
+                <a
+                  class="button"
+                  href="#all-products"
+                >
+                  View all
+                </a>
+              </div>
+            </div>
+          </section>
+        </div>
+
+        <section
+          class="section about-us"
+          id="about-us"
+        >
+          <div class="about-us__image"></div>
+
+          <article class="about-us__content">
+            <div class="container">
+              <div class="about-us__content-wrapper">
+                <h2 class="about-us__title">
+                  Timeless, for 50 years and counting
+                </h2>
+
+                <p class="about-us__text">
+                  Bang & Olufsen is expanding its Recreated Classics Program
+                  with the launch of Beosystems - a limited edition music system
+                  that transcends time by bridging the gap between one of our
+                  iconic designs from 1972 and today's cutting-edge digital
+                  technology.
+                </p>
+              </div>
+            </div>
+          </article>
+        </section>
+      </div>
+    </main>
+
+    <footer class="footer">
+      <div class="container">
+        <div class="footer__wrapper">
+          <h2 class="footer__title">Contact us</h2>
+
+          <div class="footer__content">
+            <section
+              class="footer__contact-us contact-us"
+              id="contact-us"
+            >
+              <form
+                action="POST"
+                class="contact-us__form"
+                onsubmit="this.reset(); return false;"
+              >
+                <label for="full-name">
+                  <input
+                    class="contact-us__input"
+                    type="text"
+                    name="full-name"
+                    id="full-name"
+                    placeholder="Name"
+                    required
+                  />
+                </label>
+
+                <label for="email">
+                  <input
+                    class="contact-us__input"
+                    type="email"
+                    name="email"
+                    id="email"
+                    placeholder="Email"
```

Comment:
Placeholder should a a valid value for the input [Reference](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md#:~:text=Placeholders%20should%20give%20examples%20of%20what%20should%20be%20added%2C%20to%20make%20the%20expected%20format%20clear)

Reply:
Done!